gen/generator.go

```diff
@@ -764,16 +762,30 @@ func getExampleCode(filePath string) []byte {
 	return content
 }
 
-// When GEN_CLASSES environment variable is set, the class metadata is retrieved from the APIC or devnet docs and stored in the meta directory.
-func getClassMetadata() {
+// When RE_GEN_CLASSES environment variable is set, the existing class metadata is retrieved from APIC or the latest devnet docs and stored in the meta directory.
+func reGenClassMetadata() {
+	_, re_gen_classes := os.LookupEnv("RE_GEN_CLASSES")
+	if re_gen_classes {
+		names := getFileNames(metaPath)
+		classNames := strings.Join(names, ",")
+		getClassMetadata(classNames)
+	}
+}
 
-	classNames := os.Getenv("GEN_CLASSES")
+// When GEN_CLASSES environment variable is set, the class metadata is retrieved from the APIC or the latest devnet docs and stored in the meta directory.
+func getClassMetadata(classNames string) {
+
+	if classNames == "" {
+		classNames = os.Getenv("GEN_CLASSES")
+	}
```

Comment:
Should we get the env var from outside the function?

Reply:
Done